### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by Geminii** [nuxt-community/sitemap-module#169] [Ideas]
Hi @NicoPennec 👋 

Do you think it could be a great idea to purpose a documentation for sitemap-module ?
If yes, i can purpose a MR in this way to have a unified documentation with all others nuxt module through nuxt-content.

What do you think ? 😀

**Comment by NicoPennec** [comment]
Hi @Geminii, you are reading my mind 😉 

I was just looking at what the other projects were doing about the docs.
I had planned to do this with `VuePress` some time ago (cf. [project plan](https://github.com/nuxt-community/sitemap-module/projects/1#card-36997800)) before the release of `@nuxt/content`.
But harmonization is of course the best idea so far.

I will really appreciate your help on the topic 🙏

**Comment by Geminii** [comment]
Hey 😀

Really nice project plan 👍 
I can help you on some subject 😀😀

I will purpose a PR for documentation to begin 😉

**Comment by Geminii** [comment]
@NicoPennec [PR available](https://github.com/nuxt-community/sitemap-module/pull/172)
